pkg/api/router.go

```diff
@@ -20,8 +20,13 @@ This module implements a entry into the OpenSDS northbound REST service.
 package api
 
 import (
+	"crypto/tls"
 	"fmt"
 	"net/http"
+	"strconv"
+	"strings"
+
+	. "github.com/opensds/opensds/pkg/utils/config"
```

Comment:
I think it's better to not treat `config` as internal package.

Reply:
Changed.